roles/nfs_external_storage/tasks/main.yml

```diff
@@ -215,7 +215,11 @@
   register: nfs_pod
   retries: 15
   delay: 10
-  until: nfs_pod.resources[0].status.phase == 'Running'
+  until:
+    - nfs_pod is defined
```

Comment:
nfs_pod should always be defined as is the register for the task, It should not be required.